docs/sources/flow/reference/components/otelcol.exporter.loadbalancing.md

```diff
@@ -264,6 +267,95 @@ Name | Type | Description
+    more spans than expected if the spans for the same trace are spread out to more than one Agent instance.
+
+<!-- Make "rate limiting" a URL to the tail sampler doc -->
+
+### otelcol.connector.spanmetrics
+All spans for a given `service.name` must go to the same spanmetrics Agent.
+* This can be done by configuring `otelcol.exporter.loadbalancing` with `routing_key = "service"`.
+* If this is not done, metrics generated from spans might be incorrect.
+  * For example, if similar spans for the same `service.name` end up on different Agent instances,
+    the two Agents will have identical metic series for calculating span latency, errors, and number of requests.
+  * When both Agents attempt to remote write the metrics to a database such as Mimir,
+    the series with clash with each other.
+  * At best this will lead to an error in the Agent and a rejected remote write. 
+  * At worst, it could lead to an inaccurate data dure to overlapping samples for the metric series.
+
+However, there are ways to scale `otelcol.connector.spanmetrics` without the need for a load balancer:
+1. Each Agent could add an attribute such as `agent.id` in order to make its series unique.
+  * A `sum by` PromQL query could be used to aggregate the metrics from different Agents.
+  * An extra `agent.id` attribute has a downside that the metrics stored in the database will have extra cardinality.
+2. Spanmetrics could be generated in the backend database instead of the Agent.
+  * For example, span metrics can be [generated][tempo-spanmetrics] in Grafana Cloud by the Tempo traces database.
+
+[tempo-spanmetrics]: https://grafana.com/docs/tempo/latest/metrics-generator/span_metrics/
+
+### otelcol.connector.servicegraph
+Unfortunately, there is generally no reliable way to scale `otelcol.connector.servicegraph` over multiple Agent instance.
+<!-- Or is there? Can we fix the issue below using a PromQL query? There may still be higher than normal cardinality though? -->
+* For `otelcol.connector.servicegraph` to work correctly, each "client" span must be paired 
+  with a "server" span in order to calculate metrics such as span duration.
+* If a "client" span goes to one Agent, but a "server" span goes to another Agent, 
+  then no single Agent will be able to pair the spans and a metric won't be generated.
+
+`otelcol.exporter.loadbalancing` can solve this problem partially if it is configured with `routing_key = "traceID"`.
+  * Each Agent will then be able to calculate service graph for each client/server pair in a trace.
+  * However, it is possible to have a span with similar "server"/"client" values 
+    in a different trace, processed by another Agent.
+  * If two different Agents, process similar "server"/"client" spans, 
+    they will generate the same service graph metric series.
+  * If the series from two Agents are the same, this will lead to issues 
+    when remote writing them in the backend database.
```

Comment:
```suggestion
  * If the series from two Agents are the same, this will lead to issues 
    when writing them in the backend database.
```
It's just writing to a DB...